pkg/BUILD.bazel

```diff
@@ -3521,3 +3521,10 @@ test_suite(
 )
 
 unused_checker(srcs = GET_X_DATA_TARGETS)
+
+config_setting(
```

Comment:
For the other `config_setting`s, we keep this stuff in `build/toolchains/BUILD.bazel`. Can you put it there please?

Reply:
Done